app/controllers/users_controller.rb

```diff
@@ -163,7 +163,7 @@ def update
   private
 
   def edit_user_params
-    params.require(:user).permit(:email, :first_name, :last_name)
+    params.require(:user).permit(:email, :first_name, :last_name, :indala_number)
```

Comment:
I think this might be left over from when you were trying to update in the same user edit form.